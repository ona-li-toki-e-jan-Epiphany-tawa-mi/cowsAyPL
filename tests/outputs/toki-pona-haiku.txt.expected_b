/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| + b kon li lon. li namako li suwi. li ka |
| ma seli.                                 |
\__________________________________________/
                                        \                  
                                         \                 
                                           ^__^            
                                           (oo)\_______    
                                           (__)\       )\/\
                                               ||----w |   
                                               ||     ||   
