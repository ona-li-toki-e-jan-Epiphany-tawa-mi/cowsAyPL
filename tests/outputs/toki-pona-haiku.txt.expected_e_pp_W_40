/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| kon li lon. li namak |
| o li suwi. li kama s |
| eli.                 |
\______________________/
                    \                  
                     \                 
                       ^__^            
                       (pp)\_______    
                       (__)\       )\/\
                           ||----w |   
                           ||     ||   
