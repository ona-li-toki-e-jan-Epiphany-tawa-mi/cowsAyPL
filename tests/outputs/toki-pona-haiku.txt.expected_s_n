/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| kon li lon. li namako li suwi. li kama seli.  |
\_______________________________________________/
                                             \                  
                                              \                 
                                                ^__^            
                                                (**)\_______    
                                                (__)\       )\/\
                                                 U  ||----w |   
                                                    ||     ||   
