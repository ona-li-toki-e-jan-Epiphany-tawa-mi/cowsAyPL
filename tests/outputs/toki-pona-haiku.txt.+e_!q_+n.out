/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| kon li lon. li namako li suwi. li kama seli.  |
\_______________________________________________/
        \
         \
           ^__^
           (!q)\_______
           (__)\       )\/\
               ||----w |
               ||     ||

